CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### citus v11.1.3 (October 14, 2022) ###
+
+* Adds support for PostgreSQL 15.0
+
+* Fixes a bug in `ALTER EXTENSION citus UPDATE`
```

Comment:
this is about the downgrade issue, right ?

Reply:
Yes it is.